<launch>
  <!-- Loop Computation -->
  <node pkg="loop_closure"
        name="laser_loop_closure"
        type="laser_loop_closure_node"
        output="screen">
    <remap from="~pose_graph_incremental" to="lamp/pose_graph" />
    <remap from="~keyed_scans" to="lamp/keyed_scans" />
    <remap from="~loop_closures" to="lamp/laser_loop_closures" />
    <!-- Use fixed covariances, rather than computed -->
    <param name="b_use_fixed_covariances" value="false" />
    <rosparam file="$(find lamp)/config/lamp_settings.yaml" subst_value="true"/>
    <rosparam file="$(find loop_closure)/config/laser_parameters.yaml" subst_value="true"/>     
    <rosparam file="$(find lamp)/config/precision_parameters.yaml" subst_value="true"/> 
  </node>

</launch>
